image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -71,33 +82,37 @@ object FileMetadataReader {
         }.toMap
     } getOrElse Map()
 
+  private def exportXmpProperties(metadata: Metadata): Map[String, String] =
+    Option(metadata.getFirstDirectoryOfType(classOf[XmpDirectory])) map { directory =>
+      directory.getXmpProperties.asScala.toMap.mapValues(nonEmptyTrimmed).collect {
+        case (key, Some(value)) => key -> value
+      }
+    } getOrElse Map()
+
   // Getty made up their own XMP namespace.
   // We're awaiting actual documentation of the properties available, so
```

Comment:
Couldn’t find the docs, only this: https://www.sno.phy.queensu.ca/~phil/exiftool/TagNames/XMP.html#GettyImages

But I suppose, the whole shebang will be repeated under `xmp` anyway, right?

Reply:
Yes, that's right.